Documentation/poison-report-format.md

```diff
@@ -0,0 +1,63 @@
+# Interpreting the poison report
+
+The poison report looks something like this:
+
+```
+<PrebuiltLeakReport>
+  <File Path="dotnet-sdk-6.0.100/sdk/6.0.0/Roslyn/bincore/runtimes/win/lib/netcoreapp3.1/System.Text.Encoding.CodePages.dll">
+    <Type>Hash</Type>
+    <Hash>9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08</Hash>
+    <Match Package="/home/dotnet-bot/build/previously-source-built/System.Text.Encoding.1.2.3.nupkg" PackageId="System.Text.Encoding" PackageVersion="1.2.3" File="lib/netcoreapp3.1/System.Text.Encoding.CodePages.dll" />
+  </File>
+  <File Path="dotnet-toolset-internal-6.0.100/Roslyn/bincore/runtimes/win/lib/netcoreapp3.1/System.Reflection.Metadata.dll">
+    <Type>AssemblyAttribute</Type>
+    <Hash>cf80cd8aed482d5d1527d7dc72fceff84e6326592848447d2dc0b0e87dfc9a90</Hash>
+  </File>
+  <File Path="Microsoft.DotNet.GenFacades.1.0.0-beta.20316.7/.poisoned">
+    <Type>NupkgFile</Type>
+    <Hash>e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855</Hash>
+  </File>
+</PrebuiltLeakReport>
+```
+
+The first thing to notice is the different `Type`s.  These can help you track down where the leak came from.  One match can have multiple types, e.g. `<Type>AssemblyAttribute,Hash</Type>`.
+
+## Failure types
+
+### AssemblyAttribute
+
+This means that we detected a custom attribute that source-build adds to each prebuilt and previously-source-built assembly before the build.  This is a *definite* leak - there will not be any false positives.  However, this may not be cause for failure - some categories of assemblies may leak without causing problems:
+
+- Localization assemblies, e.g. .../langcode/Assembly.Resources.dll
+- Reference assemblies (these should not ship, but also will very rarely cause problems).
+- Test or resource assemblies - these should be removed.
+
+### Hash
+
+Hash failures mean that a file matches the catalog of hashes that we take of existing files before the build.  These are commonly false positives due to:
```

Comment:
```suggestion
Hash failures mean that a file matches the catalog of hashes that we take of existing files before the build.  These are commonly false positives due to shipping static files not produced by the build such as:
```

I find it more meaningful to explain the high level view before listing examples and then the general explanation at the end.